packages/blockchain-ui/src/components/elements/ButtonList/ButtonList.tsx

```diff
@@ -1,33 +1,53 @@
 import React, { Component } from 'react';
 import './ButtonList.scss';
 import { Button } from 'carbon-components-react';
+import IPackageRegistryEntry from '../../../interfaces/IPackageRegistryEntry';
 
-interface IProps {
-    buttons: any[];
+interface ButtonListProps {
+    onProgressChange: any;
+    currentIndex: number;
+    selectedPackage: IPackageRegistryEntry | undefined;
```

Comment:
this feels wrong that you are passing data through the button list.

Reply:
Fixed - we now determine whether the Next button should be disabled in the DeployPage.